### ISSUE TITLE
Query regarding training of the Shape Captioning Network.

### ISSUE THREAD

**Issue by shikhar2333** [compsciencelab/ligdream#6]
In the paper titled "From Target to Drug: Generative Modeling for the Multimodal Structure-Based Ligand Design", for training the shape caption network, no conditional input was being passed to the VAE network since the Shape Generation Network gave only the ligand shapes as output.

So, I tried using the models provided in that paper and trained for 10k iterations with same hyperparameters as mentioned in the shape captioning network.

What I found was the shape caption network gave a single molecule as output, even on changing the seed molecule. But when I incorporated the conditional input in the VAE, I was able to capture different molecules on changing the seed molecule.

Can you point to what I might be doing wrong when training the Shape Captioning Network(when no conditional input is passed to the VAE)?
